packages/gitmoji-changelog-cli/src/index.js

```diff
@@ -2,6 +2,9 @@
 
 const yargs = require('yargs')
 const { main } = require('./cli')
+const { logger } = require('@gitmoji-changelog/core')
+
+logger.start('gitmoji-changelog v0.1.1')
```

Comment:
You could import `version` from `package.json` ?

Reply:
Good catch 👍 I wanted to do it but I forgot